addons/index.md

```diff
@@ -7,16 +7,52 @@ layout: documentation
 Add-ons can be easily installed through the Paper UI (menu entry "Add-ons").
 
+| Add-on Type                             | Description                                                                                                               |
+|-----------------------------------------|---------------------------------------------------------------------------------------------------------------------------|
+| [Bindings](bindings.html)               | Bindings integrate physical hardware, external systems and web services in openHAB                                        |
+| [User Interfaces](uis.html)             | User interfaces are either native smartphone apps or web applications that access the openHAB server through the REST API |
+| [Persistence](persistence.html)         | Persistence services allow openHAB to store time series data for history-based actions or statistics                      |
+| [Actions](actions.html)                 | Actions are predefined methods for openHAB rules and scripts                                                              |
+| [Transformations](transformations.html) | Transformations are used to translate between technical and human-readable values for items                               |
+| [Voice Services](voices.html)           | Services that provide voice enabling features, such as text-to-speech, speech-to-text etc.                                |
+| [3rd Party System Integration](io.html) | Expose openHAB to external systems                                                                                        |
 
-| Add-on Type | Description |
-|-------------|----------------------|
-| [Bindings](bindings.html) | Bindings integrate physical hardware, external systems and web services in openHAB |
-| [User Interfaces](uis.html) | User interfaces are either native smartphone apps or web applications that access the openHAB server through the REST API |
-| [Persistence](persistence.html) | Persistence services allow openHAB to store time series data for history-based actions or statistics |
-| [Actions](actions.html) | Actions are predefined methods for openHAB rules and scripts |
-| [Transformations](transformations.html) | Transformations are used to translate between technical and human-readable values for items |
-| [Voice Services](voices.html) | Services that provide voice enabling features, such as text-to-speech, speech-to-text etc. |
-| [3rd Party System Integration](io.html) | Expose openHAB to external systems |
+## Installation of Add-ons
+
+Add-ons can be installed in different ways.
+
+### Through Paper UI
+
+Navigate to the add-ons section.
+Search for the needed add-on in the categories and press install.
+
+![installing Add-ons through Paper UI](images/index_installation_paperui.gif){: width="720px"}
+
+### From the Karaf Console
+
+Connect to the device which is running openHAB and ssh into the karaf console.
+Use the following command to install a specific add-on:
```

Comment:
Please link
* https://docs.openhab.org/administration/console.html and
* https://docs.openhab.org/administration/bundles.html

The second link should actually contain that information as well...

Consider "Use the `bundle` command family to manage add-ons:"